app/views/layouts/application.html.erb

```diff
@@ -20,12 +20,15 @@
           </div>
 
           <div class="right-nav">
-            <% if session[:user_id] != nil %>
+            <% if session[:user_id] %>
               <li><%= link_to "My Account", user_path(session[:user_id]) %></li>
             <% end %>
             <li><%= link_to "Shopping Cart", cart_path %></li> 
-            <li><%= link_to "Login", github_login_path, class: "btn btn-info" %></li>
-            <li><%= link_to "Logout", logout_path, class: "btn btn-info" , method: :delete %></li>
+            <% if session[:user_id] %>
```

Comment:
should this not be "if !session[:user_id]" or "if session[:user_id] == nil"?

Reply:
Oopsy, yes absolutely!